### ISSUE TITLE
I had it working before... what happen... pip error? Help! [Fixed]

### ISSUE THREAD

**Issue by jdroo** [LykosAI/StabilityMatrix#419]
I had this working and everything the first time i installed it, but I had an issue with it hogging up my disk space, cause i went obsessive and downloaded everything shiny ... So i deleted the Data folder, and went to start over, now i get an error.

```
Unpacking resources
Unpacking resources
Error: System.IO.FileNotFoundException: pip not found
File name: 'C:\Ai\StabilityMatrix\DATA\Assets\Python310\Scripts\pip.exe'
   at StabilityMatrix.Core.Python.PyRunner.InstallPackage(String package)
   at StabilityMatrix.Core.Models.PackageModification.SetupPrerequisitesStep.ExecuteAsync(IProgress`1 progress)
   at StabilityMatrix.Core.Models.PackageModification.PackageModificationRunner.ExecuteSteps(IReadOnlyList`1 steps)
```

i checked %appdata% for anything Stability Matrix and manually removed it hoping this might fix the issue, but no dice...

help? pip worked before.... I don't get it...

**Comment by jdroo** [comment]
https://i.imgur.com/5omrFlJ.png

**Comment by jdroo** [comment]
So I...
Deleted any folders that had Anything to do with Stable diffusion and pip in my %appdata% folders
Removed all folders of pip from %appdata% Folders and anywhere else i could find em.
Removed all versions of python.
Reinstalled Python to PATH again!
Opened CMD and Installed the recommended version of pip again!
Ran the StabilityMatrix.exe
It gave me the popup to agree to junk. [was not doing that before] I agreed and continued.
I ignored the first startup window giving the list of packages, and went directly to packages [unnecessary but that's what I did]
I picked Automatic1111, on it's default settings and clicked install.

and well

`Requirement already satisfied: pip in c:\ai\stabilitymatrix\data\packages\stable diffusion webui\venv\lib\site-packages (23.3.2)`

it worked? So I got it.... That was a pain in my butt. lol
